src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4638,4 +4638,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ConstantStringTooLong" xml:space="preserve">
     <value>Length of String constant exceeds current memory limit.  Try splitting the string into multiple constants.</value>
   </data>
+  <data name="ERR_LoadDirectiveOnlyAllowedInScripts" xml:space="preserve">
+    <value>#load is only allowed in scripts</value>
```

Comment:
The intent is to say ```(SourceCodeKind.Script || SourceCodeKind.Interactive)```.  Elsewhere (in messages related to ```#r```), we used "script" (lower case) to attempt to convey this meaning.  Can anyone think of a simple way we can say this?  (we should use a single word or phrase consistently in the UI)